app/src/main/java/com/teamagam/gimelgimel/app/map/actions/timeplay/TimeplayViewModel.java

```diff
@@ -147,11 +169,30 @@ public void onMapReady() {
     showSnapshot(EARLIEST_TIMESTAMP);
   }
 
-  private String format(DateFormat dateFormat, String defaultValue) {
-    if (mCurrentDisplayedDate == null) {
-      return defaultValue;
+  private void pauseIfPlaying() {
```

Comment:
pause is enough.
Obviously, pausing an already paused process shouldn't have any effect.

Reply:
done